### DISCUSSION TITLE
Why is the official Next.js starter template so incredibly slow?

### DISCUSSION THREAD

**Discussion by stelcodes** [vercel/next.js#72364] [Help] [answered]
### Summary

I'm starting my first Next.js project and I'm shocked by how slow it is out of the box.

I'm simply creating a Next app with the recommended commands:
```
npx create-next-app@latest
npm run dev
```
- Open http://localhost:3000 after `Ready in <x> ms` prints to the console.
- Wait and wait and wait for about **30 seconds**.
- Page response completes and starter template site is shown.

I'm using a M1 Macbook Air. Tried with and without Turbopack and experience is very similar.

Similarly, when I change text within a JSX element, it takes about **20 seconds** to reload and show the changes on the browser.

What is going on??

### Additional information
Node version: v22.4.1

`package.json`:
```json
{
  "name": "dev-homepage",
  "version": "0.1.0",
  "private": true,
  "scripts": {
    "dev": "next dev",
    "build": "next build",
    "start": "next start",
    "lint": "next lint"
  },
  "dependencies": {
    "react": "19.0.0-rc-02c0e824-20241028",
    "react-dom": "19.0.0-rc-02c0e824-20241028",
    "next": "15.0.2"
  },
  "devDependencies": {
    "typescript": "^5",
    "@types/node": "^20",
    "@types/react": "^18",
    "@types/react-dom": "^18",
    "eslint": "^8",
    "eslint-config-next": "15.0.2"
  }
}

```

**Comment by icyJoseph** [comment]
Something ought to be messing it up on your system.

```console
   ▲ Next.js 15.0.3-canary.6
   - Local:        http://localhost:3000

 ✓ Starting...
 ✓ Ready in 922ms
 ○ Compiling / ...
 ✓ Compiled / in 842ms (674 modules)
 ✓ Compiled in 224ms (294 modules)
 GET / 200 in 1109ms
 ✓ Compiled /favicon.ico in 354ms (395 modules)
 GET /favicon.ico 200 in 405ms
```

I am even running Next.js off my own local repository, and it goes smooth. Using a 16Gb RAM M1 mac.

**Comment by stelcodes** [comment, marked as answer]
Ah I've found the issue. This only happens with Firefox. Very strange.
